docs/guides/becoming-a-cna-as-an-open-source-org-or-project.md

```diff
@@ -0,0 +1,207 @@
+# Becoming a CNA as an Open Source Organization or Project
+
+> **NOTE:**
+> This document was drafted using the following document revisions:
+> 
+> * CVE Numbering Authority (CNA) Rules, Version 3.0
+> * CVE Record Dispute Policy, Version 1.0
+> * CVE Program Policy and Procedure for End of Life Products, Version 1.2
+> * CVE Program Policy and Procedure for Inactive CNAs, Version 1.2
+
+## Audience and Overview
+
+[CVE Numbering Authorities](https://www.cve.org/PartnerInformation/ListofPartners) (CNAs) are organizations responsible for the regular assignment of CVE IDs to vulnerabilities, and for creating and publishing information about the Vulnerability in the associated CVE Record. You can learn more about CNAs from [the CVE website](https://www.cve.org/PartnerInformation/Partner). There are videos available covering the [CVE Program overview](https://youtu.be/rrNYEUNsXOY) and the [general process to become a CNA](https://youtu.be/13b5cuZR7CQ).
+
+This guide was written for Open Source organizations and projects that are interested in becoming a CNA and managing their own CVEs. This is not a guide for becoming a [Root](https://www.cve.org/ResourcesSupport/AllResources/CNARules#section_3_roots) which has additional requirements.
+
+This document will help cover the following topics:
+
+* Benefits of becoming a CNA as an Open Source project
+* Does your organization or project meet the requirements to become a CNA?
+* What is the process for becoming a CNA?
+
+A future document will cover other topics about CNA operations.
+
+## Why become a CNA?
+
+CVEs are a common way to record and share details about vulnerabilities in software projects. Because of their near-ubiquity, CVEs are often the method that downstream consumers will use to know when software they're using is vulnerable and requires remediation. This is especially evident with their use of security scanners and tools that report findings discovered in software and systems using the CVE identifiers.
+
+CVE Numbering Authorities (CNAs) are the entities that can allocate CVE IDs and create corresponding CVE Records within a particular scope (a product, a project or group of affiliated projects, and the like). Becoming a CNA means that your organization will have a defined scope and will be able to create CVE IDs and Records for that defined scope autonomously.
+
+Below are some of the benefits of becoming a CNA:
```

Comment:
This text seems to suggest that there are *ONLY* advantages to becoming a CNA. I don't believe that; everything has its pros and cons.

A short list of contraindicators might be wise. E.g.:

**When to not become a CNA**

Becoming a CNA adds a new commitment. In particular, don't become a CNA if:


* You don't have the time or knowledge necessary to implement CNA processes. It takes time to become a CNA, and being a CNA implies that you will process and manage vulnerability reports.
* You don't plan to issue CVEs. The whole point of becoming a CNA is to issue CVEs.

CVEs can be issues on open source software (OSS) projects by those who are not CNAs. Some CNAs already cover many OSS projects, including GitHub and Red Hat. In addition, it's always possible to file for a CVE through the "CNA of last resort" (MITRE at the time of this writing).

Reply:
Thanks David, I've added a list that captures your suggestion :)

Reply:
Thanks for the added list.

I worry that the list might not be noticed. I strongly urge adding a header before the new list, e.g., "Important Considerations" or something else. That way, people can find that list easily and are more likely to notice it when skimming.

Reply:
Done!